content/blog/2022-06-14-DVC-VS-Code-extension.md

```diff
@@ -0,0 +1,155 @@
+---
+title: >
+  Turn Visual Studio Code into an ML experimentation platform with the DVC
+  extension
+date: 2022-06-14
+description: >
+  Today we are releasing the DVC extension, which brings a full ML
+  experimentation platform to VS Code.
+descriptionLong: >
+  Today we are releasing the DVC extension, which brings a full ML
+  experimentation platform to VS Code. Manage your data, run experiments,
+  compare metrics, and visualize plots, all from the comfort of your IDE.
+picture: 2022-06-14/header.png
+author: rob_dewit
+# commentsUrl:
+tags:
+  - VS Code
+  - DVC
+  - MLOps
+  - Release
+---
+
+Since its beta release in 2017, DVC has become an essential tool for many data
+science teams. Its data versioning capabilities, reproducible pipelines, and
+experiment tracking features are at the core of our ecosystem of open MLOps
+tools.
+
+Today we are proud to launch a new product that extends how ML teams can use
+DVC: our extension for Visual Studio Code.
+
+With this extension you get a full VS Code-native experimentation platform for
+your machine learning projects. You can control datasets, run experiments, view
+metrics, create plots, and much more. All in one place in your IDE.
+
+[Install the DVC extension from the marketplace to get started.](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc)
+
+# Why a VS Code extension?
+
+We built DVC to expand upon the Git workflow. This approach has made it a
+powerful tool that can easily be picked up by users familiar with Git. However,
+a terminal does not always provide the most accessible user experience.
+Moreover, while DVC provides many useful visualizations, a CLI tool can only
+take things so far.
```

Comment:
my 2 cents - this is a good place to link to something cool and geeky (and unrelated) like:
https://github.com/aristocratos/bashtop
or
http://ninvaders.sourceforge.net/
or even
https://github.com/Swordfish90/cool-retro-term
(or... whatever 😄  )

Reply:
Or Dwarf Fortress... 😛

I'm not sure, to be honest. I like the idea, but it runs at the risk of distracting from the main point of this section.

Reply:
I see those things on a lot of blog posts, trying to spice things up a bit by nerd-sniping (which seems to fit our internal culture) - but, up to you of course 😄